prepare-python

```diff
@@ -4,7 +4,4 @@
 
 git clone https://github.com/kaitai-io/kaitai_struct_python_runtime "$PYTHON_RUNTIME_DIR"
 
-pip install enum34
-
-# this package dropped support for Python 2.7 since 3.0.0, so we're using the latest compatible version
-pip install "unittest-xml-reporting~=2.5.2"
+python -m pip install enum34 unittest-xml-reporting
```

Comment:
So we're dropping support for Python 2.7 here? Please state it in the PR message, if it was the intent.

Reply:
No - `python -m pip install unittest-xml-reporting` now works correctly on Python 2.7 as well (I tested). Quoting my commit message from 790df3c440c08d5c6cfb5cd10a3485f38ee8195b:

```
Remove no longer needed version constraint for unittest-xml-reporting
790df3c

The package now has correct Python version compatibility metadata,
so pip will automatically install the latest version compatible with
the current Python version (i. e. 2.5.2 on Python 2.7 and 3.4, and the
latest available version everywhere else) without a manual version
constraint.
```

Reply:
When you're also updating the Construct target, would you also simplify `prepare-construct` the same way?

https://github.com/kaitai-io/ci_targets/blob/9a4fe530890ebe1f4c112a7cbe9e89bb5bd0f64b/prepare-construct#L7-L8

Reply:
Ah, I didn't see that Construct has a separate prepare script - will update that as well.